.github/pull_request_template.md

```diff
@@ -1,21 +1,38 @@
 ---
 name: Pull Request
-about: A simple pull request template.
-issue: Issue # if applicable
+about: Request to pull code changes into the repository.
+title: ''
+assignees: ''
 
 ---
 
-Provide a simple description of what the PR achieves. 
+## Relevant Issues
+<!-- List relevant issue tickets here. -->
+<!-- Say "Closes #0" for issues that the PR resolves, replacing 0 with the issue number. -->
 
-## Pull Request Details
 
-Please ensure your PR fulfills the following criteria:
+## Description
+<!-- Describe how you've implemented your changes. -->
 
-- [ ] Have you joined the [PythonDiscord Community](https://pythondiscord.com/invite)?
-- [ ] Were your changes made in a Pipenv environment?
-- [ ] Does flake8 pass (```pipenv run lint```)
 
+## Reasoning
+<!-- Outline the reasoning for how it's been implemented. -->
 
-## Additional information
 
-Provide any additional information or clarifications here.
\ No newline at end of file
+## Screenshots
+<!-- Remove this section if the changes don't impact anything user-facing. -->
+<!-- You can add images by just copy pasting them directly in the editor. -->
+
+
+## Additional Details
+<!-- Delete this section if not applicable. -->
+
+
+## Did you:
+<!-- These are required when contributing. -->
+<!-- Replace [ ] with [x] to mark items as done. -->
+
```

Comment:
This line break doesn’t match the others templates

Reply:
It's there for readability, as there's more than 2 items in the list to go through and there's more than one comment involved. The extra line does not render, so it's purely for the benefit of the PR creator.